WordPress/src/main/java/org/wordpress/android/ui/jpfullplugininstall/onboarding/JetpackFullPluginInstallOnboardingDialogFragment.kt

```diff
@@ -59,12 +61,13 @@ class JetpackFullPluginInstallOnboardingDialogFragment : DialogFragment() {
     }
 
     override fun onCreateDialog(savedInstanceState: Bundle?): Dialog =
-        object : Dialog(requireContext(), theme) {
-            override fun onBackPressed() {
+        super.onCreateDialog(savedInstanceState).apply {
+            val componentDialog = dialog as ComponentDialog
+            componentDialog.onBackPressedDispatcher.addCallback(this@JetpackFullPluginInstallOnboardingDialogFragment) {
                 viewModel.onDismissScreenClick()
-                super.onBackPressed()
+                isEnabled = false
+                componentDialog.onBackPressedDispatcher.onBackPressed()
```

Comment:
Suggestion (💡): Consider updating this to the equivalent change you did on `FeatureIntroductionDialogFragment`, see below:

```
(this as ComponentDialog).onBackPressedDispatcher.addCallback(this@JetpackFullPluginInstallOnboardingDialogFragment) {
    viewModel.onDismissScreenClick()
    isEnabled = false
    onBackPressedDispatcher.onBackPressed()
}
```

Reply:
There was also another issue here that caused a crash. I tried to cast `dialog` to `ComponentDialog`, but `dialog` is null in `onCreateDialog()` since it hasn't been created yet.
Fixed: 1808fd4a471d71b87f598ddbc4c4e6f9aa2d64dc

Reply:
Nice, thanks for the fix @irfano ! 🥇